doc/manuals/sv.po

```diff
@@ -0,0 +1,3867 @@
+
+#. type: Plain text
+#: src/auth/basic/LDAP/basic_ldap_auth.8:124
+msgid "Search scope when performing user DN searches specified by the B<-f> option. Defaults to B<sub>"
+msgstr "Sökomfång vid sökning efter användar-DN som anges av alternativet B<-f>. Standardvärdet är B<sub>"
+
+#. type: Plain text
+#: src/auth/basic/LDAP/basic_ldap_auth.8:140
+msgid "The DN and password to bind as while performing searches. Required by the B<-f> flag if the directory does not allow anonymous searches."
+msgstr "DN och lösenord som ska bindas vid sökningar. Krävs av flaggan B<-f> om katalogen inte tillåter anonyma sökningar."
+
+#. type: Plain text
+#: src/auth/basic/LDAP/basic_ldap_auth.8:145
+msgid "As the password needs to be printed in plain text in your Squid configuration it is strongly recommended to use a account with minimal associated privileges.  This to limit the damage in case someone could get hold of a copy of your Squid configuration file."
+msgstr "Eftersom lösenordet måste skrivas ut i klartext i din Squid-konfiguration rekommenderas det starkt att använda ett konto med minimala behörigheter. Detta för att begränsa skadan om någon skulle få tag på en kopia av din Squid-konfigurationsfil."
+
+#. type: Plain text
+#: src/auth/basic/LDAP/basic_ldap_auth.8:163
+msgid "Use a persistent LDAP connection. Normally the LDAP connection is only open while validating a username to preserve resources at the LDAP server. This option causes the LDAP connection to be kept open, allowing it to be reused for further user validations. Recommended for larger installations."
+msgstr "Använd en permanent LDAP-anslutning. Normalt är LDAP-anslutningen endast öppen under valideringen av ett användarnamn för att spara resurser på LDAP-servern. Detta alternativ gör att LDAP-anslutningen hålls öppen, vilket gör att den kan återanvändas för ytterligare användarvalideringar. Rekommenderas för större installationer."
+
+#. type: Plain text
+#: src/auth/basic/LDAP/basic_ldap_auth.8:176
+msgid "Only bind once per LDAP connection. Some LDAP servers do not allow re-binding as another user after a successful I<ldap_bind.> The use of this option always opens a new connection for each login attempt. If combined with the B<-P> option for persistent LDAP connection then the connection used for searching for the user DN is kept persistent but a new connection is opened to verify each users password once the DN is found."
+msgstr "Bind endast en gång per LDAP-anslutning. Vissa LDAP-servrar tillåter inte ombindning som en annan användare efter en lyckad I-<ldap_bind.> Användningen av detta alternativ öppnar alltid en ny anslutning för varje inloggningsförsök. Om det kombineras med alternativet B-<-P> för beständig LDAP-anslutning, behålls anslutningen som används för att söka efter användarens DN, men en ny anslutning öppnas för att verifiera varje användares lösenord när DN har hittats."
```

Comment:
This italic markup (`I<text>`) syntax is having the same issues with turning into `I-<text>`. Please fix this and any others.